### DISCUSSION TITLE
how to run example on ubuntu (arm)?

### DISCUSSION THREAD

**Discussion by wangsirun** [cilium/ebpf#997] [Q&A]
my env is Linux titan-ubuntu1 4.9.140 #14 SMP PREEMPT Thu Jan 6 11:24:58 CST 2022 aarch64 aarch64 aarch64 GNU/Linux, when i make, the docker image repo is build on amd, so i want to know how to get the docker image of aarch64 os,please help me, thank you.

**Comment by ti-mo** [comment]
@wangsirun `make` rebuilds all testdata, it doesn't run the examples. Please follow the instructions at the bottom of the readme: https://github.com/cilium/ebpf/tree/master/examples.

Note that BPF support might be very limited on 4.9 aarch64, so your mileage may vary. Good luck!
